### DISCUSSION TITLE
Патч для JLE16_T1

### DISCUSSION THREAD

**Discussion by ghost** [Arkady23/V380-Web#62] [Ideas]
Здравствуйте, есть возможность сделать патч для камеры с данными:
{"soft_id":"JLE16_T1","dev_id":69825839,"version":1040505,"timestamp":1689713753}
Версия ПО: F_E16T1_V1.4.5.5_20221123
Версия прошивки: MVT9603S-XHR_20211208
? Заранее спасибо.

**Comment by Arkady23** [comment]
Для такой камеры у меня патча нет.

**Comment by Arkady23** [comment]
Времени много прошло, возможно вашу камеру сейчас можно оживить.
